aptos-move/framework/aptos-framework/sources/vesting.move

```diff
@@ -1252,9 +1252,7 @@ module aptos_framework::vesting {
 
         vector::for_each_ref(accounts, |addr| {
             let addr: address = *addr;
-            if (!account::exists_at(addr)) {
```

Comment:
Why delete this?

Reply:
This PR is based on https://github.com/aptos-labs/aptos-core/pull/15891 and turn on the feature to make sure all the tests still pass.

Reply:
I know but why does exists_at no longer work right in this test?

Reply:
I changed the implmentation of `exists_at`, it will already return `true` if we allow stateless account. Then here we will not create the resources needed for testing purpose.